packages/api/src/command/medical/patient/get-hl7v2-subscribers.ts

```
@@ -0,0 +1,148 @@
+    throw new Error(msg, { cause: error });
+  }
+}
+
+function combineStatesIntoReplacementObject(states: USState[]): string {
+  return `{${states.join(",")}}`;
+}
+
+function transformSubscriptionsToObject(
+  subscriptions: Hl7v2Subscription[]
+): Record<string, boolean> {
+  return subscriptions.reduce((acc, subscription) => ({ ...acc, [subscription]: true }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)